frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -36,6 +37,12 @@ type OptionalAttributes = {
   fast?: boolean;
   outerClassName?: string;
   displayErrorTooltips?: boolean;
+  /** style to pass down to the FastInput or Input */
+  style?: CSSProperties;
+  /** pass options for typeahead component */
+  options?: string[];
+  /** autocomplete flag to determine whether to use typeahed or not */
```

Comment:
really small thing but you did spell "typeahead" wrong in the comment

Reply:
lol darn it, thank you!